code/modules/mob/living/silicon/mommi/mommi_modules.dm

```diff
@@ -31,10 +31,9 @@
 	modules += new /obj/item/device/silicate_sprayer(src)
 	modules += new /obj/item/borg/fire_shield
 
-	var/obj/item/stack/cable_coil/W = new /obj/item/stack/cable_coil/yellow(src)
-	W.amount = MOMMI_MAX_COIL
-	W.max_amount = MOMMI_MAX_COIL
-	modules += W
+	respawnables = list(/obj/item/stack/cable_coil/yellow)
+	respawnables_max_amount = ENGINEERING_MAX_COIL
```

Comment:
MOMMI_MAX_COIL

Reply:
Removed this, was redundant to base definition